### PR TITLE
[Language Question Answering - Generate] Fix result, maxQACount and examples 

### PR DESCRIPTION
**Pull request by sahithikkss**
# Data Plane API - Pull Request

## API Info: The Basics
Most of the information about your service should be captured in the issue that serves as your [*engagement record*](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/271/Azure-REST-API-Stewardship?anchor=rest-api-stewardship-process).

* Link to engagement record issue:

Is this review for (select one):

- [ ] a private preview
- [x] a public preview
- [ ] GA release

### Change Scope
<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous Open API document (swagger) if applicable, and the root paths that have been updated. </sup>
* Design Document:
* Previous Open API Doc:
* Updated paths:

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azapi) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azapi).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling
 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://aka.ms/style)
 * [Open API Hub](https://aka.ms/openapiportal)

### Guidelines & Specifications
 * [Azure REST API Guidelines](https://aka.ms/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links
 * [Azure DevTools Wiki](https://aka.ms/azapi)

</details>

### CODE REVIEW

File: dev/cognitiveservices/data-plane/Language/examples/questionanswering/generate/SuccessfulSubmitQAGenerationJob.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
           "language": "en"
```
Comment (sush2218) [inline dev/cognitiveservices/data-plane/Language/examples/questionanswering/generate/SuccessfulSubmitQAGenerationJob.json:14]: use <answer>

File: dev/cognitiveservices/data-plane/Language/examples/questionanswering/generate/SuccessfulSubmitQAGenerationJob_useMarkedAnswer.json [hunk 2]
```diff
@@ -14,8 +14,8 @@
           "language": "en"
```
Comment (sush2218) [inline dev/cognitiveservices/data-plane/Language/examples/questionanswering/generate/SuccessfulSubmitQAGenerationJob_useMarkedAnswer.json:14]: user "< answer >"

**Comment by sahithikkss** [comment]
@jianyexi , could you please approve and merge as well?

**Comment by sahithikkss** [comment]
Following up on PR #20326, to fix #20052
@heaths , kindly help with review and merge

File: dev/cognitiveservices/data-plane/Language/questionanswering-generate.json [hunk 3]
```diff
@@ -284,10 +284,19 @@
...
+          "$ref": "#/definitions/GenerationJobResult"
         }
       ]
     },
+    "GenerationJobResult": {
+      "description": "Generation Job result represents the job result with extracted documents and question answers generated.",
+      "type": "object",
+      "properties": {
+        "result": {          
+          "$ref": "#/definitions/GenerationResult"
```
Comment (kristapratico) [inline dev/cognitiveservices/data-plane/Language/questionanswering-generate.json:296]: GenerationResult.documents is referencing a type that doesn't seem to exist in this file. Should `Document` -> `GenerationDocument` on L308?

Comment (sahithikkss) [reply dev/cognitiveservices/data-plane/Language/questionanswering-generate.json:296]: Correct @kristapratico . Thanks.

**Comment by sahithikkss** [comment]
@jianyexi , could you please review?
@heaths , please help with merging this PR.

**Review by heaths — APPROVED** [review]

**Comment by heaths** [comment]
The only validation error is the .NET SDK generation erring on 2 different auth schemes. Merging.

**Comment by kristapratico** [comment]
> The only validation error is the .NET SDK generation erring on 2 different auth schemes. Merging.

@heaths is there a way to fix this / open an issue somewhere? That validation error seems to be blocking real errors: https://github.com/Azure/azure-rest-api-specs/pull/20679#discussion_r968629865

**Comment by heaths** [comment]
I reported it internally, but that was a while ago. I just opened https://github.com/Azure/azure-sdk-tools/issues/4121

**Pull request merged**